packages/slice-machine/src/domain/fields.ts

```diff
@@ -196,6 +197,18 @@ export const UIDField: UIDField = {
   type: "UID",
 };
 
+export const UIDFieldLabelSchema = z.string().max(35).min(1);
+
+export const UIDFieldCustomErrorMap: z.ZodErrorMap = (issue, ctx) => {
+  if (issue.code === z.ZodIssueCode.too_big) {
+    return { message: `String is too long. Max: ${issue.maximum}` };
+  }
+  if (issue.code === z.ZodIssueCode.too_small) {
+    return { message: `This field is required` };
+  }
+  return { message: ctx.defaultError };
+};
+
```

Comment:
I wasn't sure where to put this piece of config, this place felt like the best but I'm happy to move it elsewhere